readme.md

```diff
@@ -1,13 +1,13 @@
-## What this file can
-Aggregate statistics of execution time analyzed using greatest tool [line_profiler](https://github.com/rkern/line_profiler).
+## What this program can do
+Aggregate statistics of execution time analyze by the greatest tool [line_profiler](https://github.com/rkern/line_profiler).
 
 ## Benefit
-- Unite multi-thread result
-- Unite repetitive result without modification to source code
+- Union multi-thread results
```

Comment:
union is noun. while verb. is required here

Reply:
I don't remember why I did this change....
Let me fix....